### ISSUE TITLE
Find another maintainer?

### ISSUE THREAD

**Issue by mdedetrich** [knutwalker/akka-stream-json#15]
We are currently being blocked by https://github.com/knutwalker/akka-stream-json/pull/14 due to binary compatibility issues and it appears that this repository is no longer being maintained, @knutwalker is it possible to find another maintainer for such reasons?

**Comment by knutwalker** [comment]
Yeah, you're right, my presence hasn't been great lately. I'm currently releasing #14, but if you're up for it, I can give you access for this repo and we can figure out a way for pushing releases?

**Comment by mdedetrich** [comment]
> Yeah, you're right, my presence hasn't been great lately. I'm currently releasing #14, but if you're up for it, I can give you access for this repo and we can figure out a way for pushing releases?

Sounds good. The implementation is fairly trivial (thanks to Jawn and Circe) so I suspect that most releases would be version bumps (like this one).

Shoot me an email with details on how to release into sonatype (I already have an account)

**Comment by akauppi** [comment]
@mdedetrich Did something happen about this?  I'm looking for something like akka-stream-circe.

---

Seems [so](https://github.com/mdedetrich/akka-streams-json). :)

@knutwalker Since it's easy to arrive on your repo via e.g. Google search (I did), would you like to write your summary of the state of the repo, preferably at the top of the README. This would be welcome for visitors to get a fast pointer to where the code base continues to flourish.

THANKS for the work you have done! Streaming JSON is a powerful tool, and should be made simple.

**Comment by mdedetrich** [comment]
@akauppi I ended up forking it at https://github.com/mdedetrich/akka-streams-json
